presto-verifier/src/main/java/com/facebook/presto/verifier/event/VerifierQueryEvent.java

```diff
@@ -113,10 +114,12 @@ public static VerifierQueryEvent skipped(
                 Optional.empty(),
                 Optional.empty(),
                 Optional.empty(),
-                new QueryInfo(
+                skipControl
+                        ? Optional.empty()
+                        : Optional.of(new QueryInfo(
                         sourceQuery.getControlConfiguration().getCatalog(),
```

Comment:
nit: is the indentation suggested by Intellij?

Reply:
It is

Reply:
Resolved by moving ternary operator to the end of the line.